jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -99,6 +99,13 @@ public interface Response extends Content.Sink
      */
     boolean isCommitted();
 
+    /**
+     * <p>Returns whether the last write has been initiated on the response.</p>
+     *
+     * @return whether the response is last
+     */
+    boolean isLast();
```

Comment:
I find this name confusing, as it overloads the meaning of the same method in `Content.Chunk` .

Maybe inverse the boolean and call the method `isSending`?

Reply:
I like that it uses "Last" as it is a hard link to the fact that it is true IFF and only last==true has been passed to write.  How about `isLastWrite` or `hasLastWrite`?  It is not `isLastWritten` because it may still be in progress.

Reply:
I like `hasLastWrite`.